c-cannons/bangazon-initial_site
# product views now accept customer ID

## Products

#### Descriptions

`/products/index` now only shows product for logged in customer
`@products = Product.where(customer_id: session[:customer_id])`

Added an hidden field to the create product form that automatically inputs logged in customer's ID.
`<%= form.hidden_field :customer_id, :value => @current_customer[:id] %>`

#### Steps to Test

Create a customer and find ID in database

Head to `/products/new` and create a product

Check Products table in database

New product should appear with current customer_id

Click `My Products` link in navbar

Should only show products with current customer_id

#### Link to Feature Ticket

https://github.com/c-cannons/bangazon-initial_site/issues/1
